### PR TITLE
test: make reference to custom permission in test data

### PR DESCRIPTION
**Pull request by jnussbaum**
The custom permission "discouraged-edge-cases" was not referenced anywhere.

### CODE REVIEW

File: testdata/xml-data/test-data-systematic.xml [hunk 1]
```diff
@@ -45,17 +45,17 @@
             <text encoding="utf8">Dies ist ein TestThing ohne Angabe von permissions</text>
         </text-prop>
         <resptr-prop name=":hasTestThing">
-            <resptr permissions="open">test_thing_1</resptr>
+            <resptr permissions="discouraged-edge-cases">test_thing_1</resptr>
```
Comment (jnussbaum) [inline testdata/xml-data/test-data-systematic.xml:48]: The custom permission "discouraged-edge-cases" was not referenced anywhere

**Review by Nora-Olivia-Ammann — APPROVED** [review]
